client/lxd_containers.go

```diff
@@ -1304,3 +1305,102 @@ func (r *ProtocolLXD) SetContainerMetadata(name string, metadata api.ImageMetada
 
 	return nil
 }
+
+// GetContainerTemplateFiles returns the list of name of a template files for a container.
```

Comment:
s/GetContainerTemplateFiles returns the list of name of a template files for a container./GetContainerTemplateFiles returns the list of names of template files for a container.
?